tikv-client/src/main/java/com/pingcap/tikv/catalog/Catalog.java

```diff
@@ -218,6 +218,13 @@ public TiTableInfo getTable(TiDBInfo database, String tableName) {
     Objects.requireNonNull(database, "database is null");
     Objects.requireNonNull(tableName, "tableName is null");
     TiTableInfo table = metaCache.getTable(database, tableName);
+
+    if (table == null) {
```

Comment:
should we also do `reloadCache` when database is null?

Reply:
done.